config/control_ur5.yaml

```diff
@@ -0,0 +1,29 @@
+arm_controller:
```

Comment:
Breaks what things? Should be a pretty straightforward change
`find . -type f -exec sed -i 's/arm_controller/husky_arm_controller/g' {} +`